src/go/rpk/pkg/adminapi/api_cluster.go

```diff
@@ -22,7 +22,9 @@ type ClusterHealthOverview struct {
 	AllNodes                  []int    `json:"all_nodes"`
 	NodesDown                 []int    `json:"nodes_down"`
 	LeaderlessPartitions      []string `json:"leaderless_partitions"`
+	LeaderlessCount           int      `json:"leaderless_count"`
```

Comment:
What happens if `rpk` is used against an older API that doens't have these values? Perhaps that is not a supported scenario?

Reply:
encoding/json doesn't fail if a field is not in the json when the json is being deserialized, so this will have a default of 0

Reply:
However, perhaps we should make this a pointer and then only print the row if the pointer is non-nil

Reply:
> However, perhaps we should make this a pointer and then only print the row if the pointer is non-nil


Yeah, that's what I was thinking. It's not a row, it's just the value between parentheses we could:

```
Leaderless partitions (-)
```

Or just print 

```
Leaderless partitions
```

If the field is nil